dpnp/tests/test_arraycreation.py

```diff
@@ -56,6 +56,33 @@ def test_error(self):
         assert_raises(TypeError, dpnp.array, x, ndmin=3.0)
 
 
+class TestAttributes:
```

Comment:
It'd be better to place tests relating to attributes of `ndarray` inside `test_ndarray.py` file